lib/screens/qr_code_scan_screen.dart

```diff
@@ -41,6 +41,9 @@ class _QrCodeScanScreenState extends State<QrCodeScanScreen> {
         (_) {
           Navigator.of(context).pushNamed(
             ChooseAmountSliderScreen.routeName,
+            arguments: Organisation(
+              name: "Mock Organisation",
+            ),
```

Comment:
Why do we need a mock name in here?

Reply:
in this code we simulate the qr code scanning for the ios simulator, bc there is no camera emulation for the ios simulators. But we need the organisation name in the following screen, so. 
please take a look on the condition above:
`//temporary code to test on iOS simulators
    if (kDebugMode && Platform.isIOS) {`

Reply:
Ah okay! I was only at the changes :-)